### PR TITLE
DP-7162 updated documentation, added examples

### PR DESCRIPTION
**Pull request by aditudorache**
## Amsterdam styled components pull request

Before opening a pull request, please ensure:

- [ ] The default export from a file matches the file name
- [ ] You have been following the guidelines, written in the [README](../docs/CONTRIBUTING.md#user-content-conventions-and-rules) file
- [ ] Your code has the necessary tests written
- [ ] You have updated the [CHANGELOG.md unreleased sections](../CHANGELOG.md#muser-content-unreleased)

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

File: packages/asc-core/README.md [hunk 1]
```diff
@@ -5,19 +5,38 @@
...
+
+To add a new theme:
+
+- add a new folder in src/theme/<new-theme-name>
+- implement the interfaces defined in src/theme/index.ts
+- export the them from the package (/src/index.ts)
+- use it in your project as parameter of the ThemeProvider
+
+```js
+import { newThemeName, ThemeProvider } from '@datapunt/asc-core'
```
Comment (chrisvanmook) [inline packages/asc-core/README.md]: Plz check how this is implemented in asc-ui. We could get the ThemeProvider from asc-ui, and pass a "themeName" prop, corresponding the alternative theme name

File: packages/asc-core/README.md [hunk 2]
```diff
@@ -5,19 +5,38 @@
...
+
 `styled` points to a re-exported version of `styled-components`, that could either be the react variant (default) or another one.
+
+## Add a new theme
+
+To add a new theme:
+
+- add a new folder in src/theme/<new-theme-name>
+- implement the interfaces defined in src/theme/index.ts
+- export the them from the package (/src/index.ts)
```
Comment (chrisvanmook) [inline packages/asc-core/README.md]: typo

File: packages/asc-ui/README.md [hunk 3]
```diff
@@ -1 +1,94 @@
...
+
+```
+
+The background color of the StyledLinkList component has the color of `tint.level5` widths smaller than `laptopM` breakpoint and `tint.level1` for widths larger than `laptopM`
+The colors and the breakpoints are defined in the `asc-core` default theme
+
+Each component exposes its style that can be used a a (class)selector in another component. For example LinkListItem has an styles.LinkListStyle selector.
+We use the exposed selector to change the color of the LinkedListItems to the `primary` theme color. In this case we don't need to create a new StyledLinkedList item to override the style.
+
+The same can be acheved by creating of a new StyledLinkedListItem:
```
Comment (chrisvanmook) [inline packages/asc-ui/README.md]: typo

File: packages/asc-ui/README.md [hunk 4]
```diff
@@ -1 +1,94 @@
...
+    <LinkListItem href="/">Item2</LinkListItem>
+  </StyledLinkList>
+</ThemeProvider>
+
+```
+
+The background color of the StyledLinkList component has the color of `tint.level5` widths smaller than `laptopM` breakpoint and `tint.level1` for widths larger than `laptopM`
+The colors and the breakpoints are defined in the `asc-core` default theme
+
+Each component exposes its style that can be used a a (class)selector in another component. For example LinkListItem has an styles.LinkListStyle selector.
```
Comment (chrisvanmook) [inline packages/asc-ui/README.md]: "a a""

File: packages/asc-core/README.md (hunk 1 above)
Comment (aditudorache) [reply packages/asc-core/README.md]: will add new example with the correct usage

**Comment by aditudorache** [comment]
Further handled in #105 Issue105-update-documentation

**Pull request closed without merging**